frontend-react/src/components/ManagePublicKey/ManagePublicKeyChooseSender.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useEffect, useState } from "react";
+import { Form, Dropdown, Button, FormGroup } from "@trussworks/react-uswds";
+
+import { useOrganizationSenders } from "../../hooks/UseOrganizationSenders";
+import Spinner from "../Spinner";
+
+export interface ManagePublicKeyChooseSenderProps {
+    onSenderSelect: (sender: string) => void;
+}
+
+export default function ManagePublicKeyChooseSender({
+    onSenderSelect,
+}: ManagePublicKeyChooseSenderProps) {
+    const { isLoading, senders } = useOrganizationSenders();
+    const [selectedSender, setSelectedSender] = useState(
+        senders?.length === 1 ? senders[0].name : ""
+    );
+
+    useEffect(() => {
+        if (senders?.length === 1) {
+            onSenderSelect(selectedSender);
+        }
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [senders]);
+
+    function handleSubmit() {
+        onSenderSelect(selectedSender);
+    }
+
+    return (
+        <div data-testid="ManagePublicKeyChooseSender">
+            {isLoading && <Spinner message="Loading..." />}
+            {!isLoading && senders && senders?.length > 1 && (
+                <Form name="sender-select" onSubmit={handleSubmit}>
+                    <FormGroup>
+                        <Dropdown
+                            id="senders-dropdown"
+                            name="senders-dropdown"
+                            onChange={(e) => {
+                                setSelectedSender(e.target.value);
+                            }}
+                        >
+                            <option value="">-Select-</option>
+                            {senders?.map(({ name }) => (
+                                <option key={name} value={name}>
+                                    {name}
+                                </option>
+                            ))}
+                        </Dropdown>
+                        <Button
+                            key="submit-sender"
+                            type="submit"
+                            outline
+                            className="padding-y-1"
```

Comment:
This'd be @jillian-hammer's call, but I think we should align with the default styles of out-of-the-box components like Button rather than apply custom styling.  If we need to create a "slim" button, we should create a new variant for that component so we don't end up with ad hoc styles all over the site.